aws-lc-fips-sys/builder/cmake_builder.rs

```diff
@@ -150,6 +150,17 @@ impl CmakeBuilder {
                 cmake_cfg.define("CMAKE_OSX_ARCHITECTURES", "arm64");
             }
         }
+
+        if target_env() == "ohos" {
+            let ndk = env::var("OHOS_NDK_HOME").expect("Try to get ohos ndk home failed.");
```

Comment:
This concern applies to the other change as well...

* Instead of panicking the build, should this be conditioned on the required environment variable being set.
```
if target_env() == "ohos" && env::var("OHOS_NDK_HOME").is_some() {
```
* If the environment already has a `CMAKE_TOOLCHAIN_FILE` set, I think that would take precedence over the `CMAKE_TOOLCHAIN_FILE` being set here. (This change will conflict with #431, so we'll need to consider this during a merge.)

Reply:
You are right, our code is more robust now. Thanks